package.json

```diff
@@ -26,8 +26,8 @@
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "build": "tsc --project .",
-    "changelog": "node ./src/cli.js"
+    "build": "tsc --project . && chmod +x 'dist/cli.js'",
```

Comment:
Could we move this to the `prepublishOnly` step instead? It's not really necessary during the build. Plus, this command does not work across all platforms. If we're to include non-cross-platform scripts, better that they be kept to the publishing flow if possible.